### PR TITLE
Fix browserify issue

### LINKED ISSUE #118: An in-range update of standard is breaking the build 🚨
## Version **11.0.0** of [standard](https://github.com/standard/standard) was just published.

<table>
  <tr>
    <th align=left>
      Branch
    </th>
    <td>
      <a href="/ethereumjs/ethereumjs-util/compare/greenkeeper%2Fstandard-11.0.0">Build failing 🚨</a>
    </td>
  </tr>
  <tr>
    <th align=left>
      Dependency
    </td>
    <td>
      standard
    </td>
  </tr>
  <tr>
    <th align=left>
      Current Version
    </td>
    <td>
      10.0.3
    </td>
  </tr>
  <tr>
    <th align=left>
      Type
    </td>
    <td>
      devDependency
    </td>
  </tr>
</table>

This version is **covered** by your **current version range** and after updating it in your project **the build failed**.

standard is a devDependency of this project. It **might not break your production code or affect downstream projects**, but probably breaks your build or test tools, which may **prevent deploying or publishing**.

<details>
<summary>Status Details</summary>

- ❌ **continuous-integration/travis-ci/push** The Travis CI build failed [Details](https://travis-ci.org/ethereumjs/ethereumjs-util/builds/343182758?utm_source=github_status&utm_medium=notification)
- ✅ **coverage/coveralls** First build on greenkeeper/standard-11.0.0 at 99.17% [Details](https://coveralls.io/builds/15582885)
</details>

---

<details>
<summary>Commits</summary>
<p>The new version differs by 50 commits.</p>
<ul>
<li><a href="https://urls.greenkeeper.io/standard/standard/commit/a5994268773a1c12d9e1e23c786ed8f7e90d41e5"><code>a599426</code></a> <code>authors</code></li>
<li><a href="https://urls.greenkeeper.io/standard/standard/commit/4b7fac5e97b09b3d102fb07db8c6ccf0fbe6c1a3"><code>4b7fac5</code></a> <code>11.0.0</code></li>
<li><a href="https://urls.greenkeeper.io/standard/standard/commit/76bf851c5e24a2a956a19c04a1d635021418506f"><code>76bf851</code></a> <code>standard-packages@3.4.0</code></li>
<li><a href="https://urls.greenkeeper.io/standard/standard/commit/22673fc7ac2dad0239e8a16660734ac4b1c50056"><code>22673fc</code></a> <code>test: add --fix flag to attempt fixing packages before failing</code></li>
<li><a href="https://urls.greenkeeper.io/standard/standard/commit/50c466cb2a287cbac365629fbd634baf2a8b27e5"><code>50c466c</code></a> <code>remove hardcoded npm run test scripts</code></li>
<li><a href="https://urls.greenkeeper.io/standard/standard/commit/cc7929058a9f22a89de85e68dcb410f70cad501e"><code>cc79290</code></a> <code>.npmignore</code></li>
<li><a href="https://urls.greenkeeper.io/standard/standard/commit/7b5e48249747fe3dc54dfa224d118b31dd9c55a9"><code>7b5e482</code></a> <code>.npmignore</code></li>
<li><a href="https://urls.greenkeeper.io/standard/standard/commit/538f087761b34cca4d9a8d5f9c4d71cf41968a07"><code>538f087</code></a> <code>standard-engine@8.0</code></li>
<li><a href="https://urls.greenkeeper.io/standard/standard/commit/01479896b434eb22ea9b5b4b87b2036ccfbccdd8"><code>0147989</code></a> <code>eslint-config-standard-jsx@5.0.0</code></li>
<li><a href="https://urls.greenkeeper.io/standard/standard/commit/7ff022fe1e5e4c81bbc6e5ac2e0849a48ab0cc63"><code>7ff022f</code></a> <code>eslint-config-standard@11.0.0</code></li>
<li><a href="https://urls.greenkeeper.io/standard/standard/commit/c9e8503a3e7e9a0117ae6aa3bae8fa7154ff3192"><code>c9e8503</code></a> <code>changelog</code></li>
<li><a href="https://urls.greenkeeper.io/standard/standard/commit/a5333eb74c28f83a98ad741886512e298cf0726c"><code>a5333eb</code></a> <code>changelog</code></li>
<li><a href="https://urls.greenkeeper.io/standard/standard/commit/98f513c65c685551cb85113dafeed541f1a75d7e"><code>98f513c</code></a> <code>test style</code></li>
<li><a href="https://urls.greenkeeper.io/standard/standard/commit/8376af6b9bb2bf45afc380bbb9cce11d5a237e7a"><code>8376af6</code></a> <code>don't peg CPU when running tests</code></li>
<li><a href="https://urls.greenkeeper.io/standard/standard/commit/47489dad065b9e5df04fc8b4d0a77f6fea97777f"><code>47489da</code></a> <code>update changelog</code></li>
</ul>
<p>There are 50 commits in total.</p>
<p>See the <a href="https://urls.greenkeeper.io/standard/standard/compare/91e80a1feba31bb1c7b1ab57283c4e3cd93d56ab...a5994268773a1c12d9e1e23c786ed8f7e90d41e5">full diff</a></p>
</details>

<details>
<summary>FAQ and help</summary>

There is a collection of [frequently asked questions](https://greenkeeper.io/faq.html). If those don’t help, you can always [ask the humans behind Greenkeeper](https://github.com/greenkeeperio/greenkeeper/issues/new).
</details>

---

Your [Greenkeeper](https://greenkeeper.io) Bot :palm_tree:

### PR DESCRIPTION
**Pull request by holgerd77**
Same as in ``ethereumjs-block``, see https://github.com/ethereumjs/ethereumjs-block/pull/40.

Also fixes #118.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,18 +17,6 @@
     "build:dist": "babel index.js --source-root ./ -d ./dist",
     "build:docs": "documentation build ./index.js --github --sort-order='alpha' -f md > ./docs/index.md"
   },
-  "browserify": {
```
Comment (Silur) [inline package.json]: maybe leave this as a script for containers etc are not forced to have a global browserify in PATH and only `npm ...` scripts can access the `.bin` dir?

Comment (holgerd77) [reply package.json]: I think this was never meant to be used for ``browserify`` itself but is just used internally for ``karma`` testing. Or am I wrong here?

Comment (alanshaw) [reply package.json]: Yes there is no browserify in the build for this module - it's only for karma

**Comment by alanshaw** [comment]
Ping! Can has merge+release plz?

**Comment by holgerd77** [comment]
@alanshaw Sorry, forgot about that one, just merged, will prepare a minor release asap.

**Comment by alanshaw** [comment]
thank you @holgerd77  ✨

**Pull request merged**